extensions/resource-deployment/src/ui/deployClusterWizard/deployClusterWizard.ts

```diff
@@ -0,0 +1,116 @@
+import { IKubeService } from '../../services/kubeService';
+import { IAzdataService } from '../../services/azdataService';
+import { DeploymentProfilePage } from './pages/deploymentProfilePage';
+import { INotebookService } from '../../services/notebookService';
+import { DeployClusterWizardModel } from './deployClusterWizardModel';
+import * as VariableNames from './constants';
+const localize = nls.loadMessageBundle();
+
+export class DeployClusterWizard extends WizardBase<DeployClusterWizard, DeployClusterWizardModel> {
+
+	public get kubeService(): IKubeService {
+		return this._kubeService;
+	}
+
+	public get azdataService(): IAzdataService {
+		return this._azdataService;
+	}
+
+	public get notebookService(): INotebookService {
+		return this._notebookService;
+	}
+
+	constructor(private wizardInfo: WizardInfo, private _kubeService: IKubeService, private _azdataService: IAzdataService, private _notebookService: INotebookService) {
+		super(DeployClusterWizard.getTitle(wizardInfo.type), new DeployClusterWizardModel(wizardInfo.type));
+	}
+
+	public get deploymentType(): BdcDeploymentType {
+		return this.wizardInfo.type;
+	}
+
+	protected initialize(): void {
+		this.setPages(this.getPages());
+		this.wizardObject.generateScriptButton.hidden = true;
+		this.wizardObject.doneButton.label = localize('deployCluster.openNotebook', 'Open Notebook');
+	}
+
+	protected onCancel(): void {
+	}
+
+	protected onOk(): void {
+		process.env[VariableNames.AdminPassword_VariableName] = this.model.getStringValue(VariableNames.AdminPassword_VariableName);
+		this.notebookService.launchNotebook(this.wizardInfo.notebook).then((notebook: azdata.nb.NotebookEditor) => {
+			notebook.edit((editBuilder: azdata.nb.NotebookEditorEdit) => {
+				editBuilder.insertCell({
+					cell_type: 'code',
+					source: this.model.getCodeCellContentForNotebook()
+				}, 7);
+			});
+		}, (error) => {
+			vscode.window.showErrorMessage(error);
+		});
+	}
+
+	private getPages(): WizardPageBase<DeployClusterWizard>[] {
+		const pages: WizardPageBase<DeployClusterWizard>[] = [];
+		switch (this.deploymentType) {
+			case BdcDeploymentType.NewAKS:
+				pages.push(
+					new DeploymentProfilePage(this),
+					new AzureSettingsPage(this),
+					new ClusterSettingsPage(this),
+					new ServiceSettingsPage(this),
+					new SummaryPage(this));
+				break;
+			case BdcDeploymentType.ExistingAKS:
+				pages.push(
+					new DeploymentProfilePage(this),
+					new TargetClusterContextPage(this),
+					new ClusterSettingsPage(this),
+					new ServiceSettingsPage(this),
+					new SummaryPage(this));
+				break;
+			case BdcDeploymentType.ExistingKubeAdm:
+				pages.push(
+					new DeploymentProfilePage(this),
+					new TargetClusterContextPage(this),
+					new ClusterSettingsPage(this),
+					new ServiceSettingsPage(this),
+					new SummaryPage(this));
+				break;
+			default:
+				throw new Error(localize('deployCluster.UnknownDeploymentTarget', "Unknow wizard type: {0}", this.deploymentType));
```

Comment:
>Unknow [](start = 71, length = 6)

Unknown

Reply:
This is still misspelled 

---
In reply to: [327735658](https://github.com/microsoft/azuredatastudio/pull/7183#discussion_r327735658) [](ancestors = 327735658)